backend/api/Models/Models.cs

```diff
@@ -18,14 +19,85 @@ public class Robot
         [MaxLength(128)]
         [Required]
         public string SerialNumber { get; set; }
+        [MaxLength(128)]
+        [Required]
+        public string Logs { get; set; }
+        [MaxLength(128)]
+        [Required]
+        public string Host { get; set; }
+        [Required]
+        public int Port { get; set; }
+        [Required]
+        public bool Enabled { get; set; }
         [Required]
         public RobotStatus Status { get; set; }
     }
 
+    public class Report
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        [Required]
+        public string Id { get; set; }
+        [Required]
+        public virtual Robot Robot { get; set; }
+        [MaxLength(128)]
+        [Required]
+        public string IsarMissionId { get; set; }
+        [MaxLength(128)]
+        [Required]
+        public string EchoMissionId { get; set; }
+        [MaxLength(128)]
+        [Required]
+        public string Log { get; set; }
+        [Required]
+        public ReportStatus ReportStatus { get; set; }
+        [Required]
+        public DateTimeOffset StartTime { get; set; }
+        public DateTimeOffset EndTime { get; set; }
+        [Required]
+        public virtual ICollection<ReportEntry> Entries { get; private set; }
+    }
+
+    public class ReportEntry
```

Comment:
This probably should be in Database/Models/ folder

Reply:
Agreed